parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py

```diff
@@ -317,7 +353,11 @@ def _harvest_entries(self, filename: str, entries_start_line: int) -> Iterator[N
                                 f"Reading error at line {line_number + 1}, unexpected format: '{self.parser_logger.ellipsis(line)}'"
                             )
                         if property_name:
-                            data.properties["prop_" + property_name + "_" + lc] = property_value
+                            prop_key = "prop_" + property_name + "_" + lc
+                            data.properties[prop_key] = property_value
+                            prop_comments = self._get_comments_above(data, line_number)
+                            if prop_comments:
+                                data.comments[prop_key + "_comments"] = prop_comments
 
         data.id = "__footer__"
         data.preceding_lines.pop(0)
```

Comment:
don't we have to do something about comments there also (it's an edge case, it's ok if you just put a TODO comment).

Reply:
Comments are still stored in the `preceding_lines` property for the footer, no changes

Reply:
ok